vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/VSImage.java

```diff
@@ -126,4 +139,10 @@ public void setCategories(List<ImageCategory> categories) {
         this.categories = categories;
     }
   
+    @Override
+    public String toString() {
+        return "VSImage{" +
+                "disableFlag='" + disableFlag +
+                '}';
+    }
```

Comment:
remove this method or make it more useful. Write now all I learn is that the image is disabled or not but nothing else. I won't even know if the object is the same as another because I don't get the hashcode.